Ssmpu/Memory_Protection_Unit.adoc

```diff
@@ -5,14 +5,16 @@ An optional RISC-V Memory Protection Unit (MPU) provides per-hart supervisor-mod
 
 Like PMP, the granularity of MPU access control settings are platform-specific and within a platform may vary by physical memory region, but the standard MPU encoding should support regions as small as four bytes. 
 
-MPU checks could be applied to all accesses for both U mode and S mode, depending on the values in the configuration registers. MPU registers can always be modified by M-mode and S-mode software. MPU registers can grant permissions to U-mode, which has none by default, and revoke permissions from S-mode, which has full permissions by default.
+MPU checks could be applied to all accesses for both U mode and S mode, depending on the values in the configuration registers.
+M-mode accesses are not affected.
+MPU registers can always be modified by M-mode and S-mode software. MPU registers can grant permissions to U-mode, which has none by default, and revoke permissions from S-mode, which has full permissions by default.
```

Comment:
let's make it clear that "full permissions" in this case is whatever M-mode allowed through PMP/ePMP.